internal/generator/vector/source/audit_logs.go

```diff
@@ -9,6 +9,8 @@ const HostAuditLogTemplate = `
 # {{.Desc}}
 [sources.{{.ComponentID}}]
 type = "file"
+file_key = "file"
+host_key = "host"
```

Comment:
where is `file` key used? `host` seems to be used in cloudwatch VRL

Reply:
yes host is required for cloudwatch but file_key is not required so taking it out.